api/tests/unit/domain/models/User_test.js

```diff
@@ -15,6 +15,8 @@ describe('Unit | Domain | Models | User', function () {
         lastTermsOfServiceValidatedAt: '2020-05-04T13:40:00.000Z',
         mustValidateTermsOfService: true,
         isAnonymous: false,
+        hasBeenAnonymised: false,
```

Comment:
alors moi si j'écoutais mon coeur je supprimerais carrément ce fichier, mais je ne veux pas pousser au pêché 🐍 🍎

Reply:
Je trouve ça pas mal les TU de modèles pour tester les méthodes de la class User, comme par exemple 
```
hasAccessToCertificationCenter(certificationCenterId) {
    return this.certificationCenterMemberships.some(
      (certificationCenterMembership) =>
        certificationCenterMembership.certificationCenter.id === certificationCenterId &&
        isNil(certificationCenterMembership.disabledAt)
    );
  }
```
Pour ma compréhension, en quoi ça ne teste rien selon vous ?
Après la test `should build a user from raw JSON` en effet, je supprimerais bien

Reply:
Oui pardon j'ai pas été claire. Je parle précisément des it de ce type, les simples passeurs de data.
Pardon pour le flou

Reply:
Ah oui, complètement d'accord alors !